.gitignore

```diff
@@ -11,3 +11,4 @@ npm-debug.log
 # for maintenance purposes - not best practices
 package-lock.json
 
```

Comment:
Don't forget to add the .env file in .gitignore. You don't want your environment variables to be public